diff --git a/media-sound/spotify/spotify-1.2.42.ebuild b/media-sound/spotify/spotify-1.2.42.ebuild
index 2a4aa0f..7bea6b1 100644
--- a/media-sound/spotify/spotify-1.2.42.ebuild
+++ b/media-sound/spotify/spotify-1.2.42.ebuild
@@ -32,7 +32,7 @@ RDEPEND="
 	media-libs/alsa-lib
 	media-libs/fontconfig
 	media-libs/harfbuzz
-	media-libs/mesa[X(+)]
+	media-libs/mesa
 	net-misc/curl[ssl]
 	net-print/cups[ssl(+)]
 	sys-apps/dbus
